### PR TITLE
qrcode plugin has a panel field now

### PR DESCRIPTION
**Pull request by bnomei**

### CODE REVIEW

File: content/plugins/bnomei/qrcode/plugin.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@ Category: utilities
 
 ----
 
-Description: Generate QRCodes
+Description: Generate QRCodes. The included Panel-Field will show the QRCode and trigger download on click.
```
Comment (lukasbestle) [inline content/plugins/bnomei/qrcode/plugin.txt]:
```suggestion
Description: Generate QRCodes. The included Panel field will show the QRCode and trigger download on click.
```

**Review by lukasbestle — CHANGES_REQUESTED** [review]
Could you please rebase your PR against the `main` branch of getkirby.com and/or squash the commits? Otherwise we'll have a mess of all the commits in the commit history.

**Comment by bnomei** [comment]
that would mean i can not do this web-based but have to download the full getkirby.com repo and do git "magic". maybe some other day. just leave the pr open if you want. 

https://medium.com/gyanblog-coding-problems/git-how-to-create-a-pr-with-no-history-of-commits-96dcfa4ea532

**Comment by lukasbestle** [comment]
I'm assuming you have created the PR on GitHub directly. You can avoid the commit history issue by directly editing the file in our `main` branch, which will automatically create a new PR branch for you in your own fork that is based on the latest state. If you reuse your own `master` branch over and over, the old already merged commits cause this issue, so please don't do that. :)

For this PR I'll be using GitHub's squash merge feature, which isn't optimal as your GPG signature on the original commit is lost. But for this change that's probably fine.

**Review by lukasbestle — APPROVED** [review]

**Comment by bnomei** [comment]
thanks for explaining  @lukasbestle

**Pull request merged**